### DISCUSSION TITLE
How to draw a terminal interface using slint？

### DISCUSSION THREAD

**Discussion by iamazy** [slint-ui/slint#6714] [Q&A]
I want to draw a terminal interface using slint, but I don't known which component I should use.
Just like this image:
![image](https://github.com/user-attachments/assets/4348cecb-bbc7-4d37-86cd-474082ddacef)

**Comment by ogoffart** [comment]
You would probably use a `Text` or `TextEdit` or `TextInput`
One thing is that Slint doesn't yet have rich text Text component (issue https://github.com/slint-ui/slint/issues/2723 ). But color can be achieved with multiple `Text` in difference color.
